### PR TITLE
Mitaka

### PR DESCRIPTION
**Pull request by pjbreaux**
@zancas @larkinkevin 
#### What's this change do?
Brings mitaka branch up-to-date with liberty changes.

#### Any background context?
There were a few conflicts in the Makefile that I fixed. Make sure any changes there look correct. @larkinkevin will come behind this merge with a PR directly into mitaka to get the Jenkins integration working.

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -61,8 +61,8 @@ export UPPER_CONSTRAINTS_FILE := https://git.openstack.org/cgit/openstack/requir
 # TLC path and python path requirements
 export PATH := /tools/bin:$(PATH)
 export PYTHONPATH := /tools/lib:/tools/bin:$(PYTHONPATH)
-export USER := buildbot
-export TEST_OPENSTACK_DISTRO := $(BRANCH)
+export USER := jenkins
+export TEST_OPENSTACK_DISTRO := liberty
```
Comment (larkinkevin) [inline systest/Makefile]: This should be "mitaka" instead of "liberty".

**Comment by pjbreaux** [comment]
@larkinkevin: address your comment on liberty/mitaka

File: systest/Makefile [hunk 2]
```diff
@@ -130,21 +130,11 @@ cleanup_failed_tlc:
...
-# We ignore the return value so that we run all of the tests
-functest: install_tlc install_test_infra
-	-$(MAKE) -C . tempest_12.1.1_undercloud
-
 # Not using the tempest TLC files for liberty because they install barbican
 # which causes the tests to lockup/hang
 
 # Tempest Tests for 12.1.x undercloud VE deployment
-tempest_12.1.1_undercloud:
+tempest_12.1.1_undercloud_vxlan:
```
Comment (larkinkevin) [inline systest/Makefile]: The "RESULTS_DIR" variable should have been changed for this job (by dropping the "$(CHANGESOURCE)" path segment).

Comment (larkinkevin) [inline systest/Makefile]: Never mind.  Looks like it's already there.

**Comment by larkinkevin** [comment]
Looks good to me.

**Comment by pjbreaux** [comment]
@larkinkevin: you have to approve the PR before we can merge.

**Comment by zancas** [comment]
This PR LGTM.   @pjbreaux @larkinkevin are y'all expecting me to merge it?

**Comment by larkinkevin** [comment]
No thanks @zancas. @pjbreaux already took care of the merge forward (this PR is the outcome of his merge from liberty to mitaka).  I guess @mattgreene will perform the official merging/accepting (in his capacity as merge czar/assignee ;0)

**Comment by larkinkevin** [comment]
@zancas, it would be most helpful if you could do for the agent PR what @pjbreaux did for this driver PR (ie. submit the merge forward from liberty to mitaka).

**Pull request merged**